pages/linux/fuser.md

```diff
@@ -1,8 +1,24 @@
 # fuser
 
 > Display process IDs currently using files or sockets.
-> Require admin privileges.
+> Requires admin privileges.
 
-- Identify process using a TCP socket:
+- Identify processes using a TCP socket:
 
 `fuser -n tcp {{port}}`
+
+- Find which processes are accessing a directory:
+
+`fuser {{path/to/directory}}`
+
+- Kill and signal processes accessing a file:
+
+`fuser -k {{path/to/file}}`
+
+- Find which processes are accessing a file:
+
+`fuser -m {{path/to/file}}`
```

Comment:
Is there an expanded form for `-m` ? It is not clear what it stands for.

Reply:
Looks like it is for `--mount`

Reply:
Hmm .. but why is -m only for file then ? I guess it can be for directory also ?

Reply:
Oh, maybe it's different per-OS?

- `name` ([Linux.die](https://linux.die.net/man/1/fuser))
- `--mount` ([DigitalOcean ref.](https://www.digitalocean.com/community/tutorials/how-to-use-the-linux-fuser-command))
- `mmapped files` ([BSD](https://www.freebsd.org/cgi/man.cgi?query=fuser))

I have no idea then...

Reply:
ping @mendesmiguel

Reply:
Looks like the `-m` option is to check for a file of a mounted filesystem, and `--mount` is just an alias. I see both `-m` and `--mount` listed on my machines (Ubuntu 18 and Debian 9). I'm not sure why linux.die.net does not list the long options, but all the three forms listed by @pxgamer actually mean the same thing (just worded differently).

Can you guys confirm about long options being present on your machines too? Looks like `--namespace`, `--kill`, `--mount` and `--verbose` are the ones to go with here.

Reply:
Yes. It also supports a directory - `If a directory file is specified, it is automatically changed to NAME/.`